### PR TITLE
Add filestash starter

### PR DESCRIPTION
**Pull request by hmmhmmhm**
Add starter links for https://www.filestash.app

### CODE REVIEW

**Comment by coffee-cup** [comment]
The filesystem for Railway deploys is ephemeral so I don't think we should have a starter that promotes using it like this.

**Comment by hmmhmmhm** [comment]
@coffee-cup Could you please check again? 🥺
it doesn't store files in the railway...!
The railway only serves as the gateway for external storage.
You can just think of it as an external service viewer that can be uploaded and downloaded
And as a viewer, it supports a lot of services. (like a S3, GDrive, FTP, MySQL)

**Comment by coffee-cup** [comment]
Ah my bad, I'll re-open it

File: examples/filestash/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
...K4Jnomd85DZJ\\nkiwS+wDYX4EJC1nnbymVj5MwTT+LuhTolKLZvsVEEc9/OiqHzIQp+wI5lDn+ZCMG\\n9qEgYTmgWGR1iXDh0iPFME1MtW470buIp6RwJWa/RdckH1kK8N2h3CbIdiJWzqKG\\n+hai9FMs4ezqitwkWEHCHEAqfazu8NMU+7Y1kk3evnlocSnxbGjzFAXKOa59zKUa\\nCyZrf6nCD2tWhgRyif854FQdyRp1ip639ugvu9pdqheNRqa2eS7txcvxZLGtcgnl\\nFfOCnsSw0i2zK4SFRxpz0pwb3ejCuEc/JgwnWUc/jy+nPyIGKkiWVJ5Kl/0cmvSt\\nsHkAEQEAAYkCNgQYAQoAIBYhBA8jju0YURm65uBRuV8B3hu/lrDzBQJgl897AhsM\\nAAoJEF8B3hu/lrDzOdAQAJ9wepmfH9qz1YNeNHXkKlc7qn7nIC4ZRqfPgY9EeW1P\\ndtBWB9BTiwqDdFxRoA68NK6Kr9p2TO79QJDpqyAPZsud7hGJBs1fQaTo03dQuzo6\\nh6G8T9X0ifcAa1U8zOBU1a4I2M230GQNBKYREm9vNZV250bNjbV/Uy03wkcRMXJE\\nOBXwBTWjQG/Yubr3KpxPeHG+3Y3mD3Cty1gk81ndncOCdFKoC4I/Yv556UbTcT2a\\nHFhk+ImP+KXBZ2pgaKGr6b8VOzg9+TMPETXYfdM4U8o7InTHbmesqsHVnp2CVTzt\\ne4Yv4mY7mDMRNKVvpZ1uu9lzob9j5LxzcQfPqrwimq+sS4FIPqLm7k1r0bI6Pyqi\\nLXX8YZgOf6Rc126K9jjsboTv+J5n0MUCKbXwbGaeBW2SFfYz9PkD5klb00AlXudZ\\nM1tflOH2fYWlmQzby7cy60CBIOdeBTcI75KyTHXI2ayP4K8HotWy57MoLyg8kfnH\\nFcYrh4GpBTPmhcJVgx8UlZP990WkZkwNVr6w6S3vZhGw0XTW9TwN9FFTfWbezL80\\nI0FS3diIV6xc3Chb1Udfuy0CDjSS/tbD3neuoYGyk8mrVZ7xn1HgB3/abISnTh52\\nWfaP6XyUf6zBSjP9O5NgrrGCfj7R/w4HytVf8J0+nnaJfjObTBJhM4KL9v10WoHh\\n=RQu+\\n-----END PGP PUBLIC KEY BLOCK-----\\n"
```
Comment (coffee-cup) [inline examples/filestash/Dockerfile:5]: I think the public key should be a variable in the template

File: examples/filestash/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+tags:
+  - s3
+  - storage
+---
+
+# Filestash
+
+> Host Web Storage Dashboard of AWS S3, Dropbox, GDrive, WebDAV, SFTP, etc. via Railway
+
+> [ ✅ IMPORTANT!] This requires adding PORT 8334 to variable.
```
Comment (coffee-cup) [inline examples/filestash/README.md:14]: Can this be added to the template?

**Pull request closed without merging**